src/main/resources/XWiki/MapMacro.xml

```diff
@@ -664,18 +680,25 @@ No content.
 = Exemples =
 
 == Simple map ==
-{{code}}
-{{map location="10, rue Pernety, France"/}}
-{{/code}}
+(% class="box"%)((( (% class="code"%)(((
+{{{ {{map location="10, rue Pernety, France"/}} }}}
+))))))
```

Comment:
I really don't see the point of this overcomplicated hack just to avoid using code macro. What is so wrong with using code macro ?

Reply:
I think the code macro has its own purpose: see the highlighting feature.